### PR TITLE
Add parameters and doc to /kml_labels.py script

### PR DESCRIPTION
**Pull request by oterral**
@loicgasser what do you think ?

### CODE REVIEW

File: scripts/kml_labels.py [hunk 1]
```diff
@@ -58,7 +59,55 @@ def escape_chars(txt):
...
+    -t <object types> : Optional. List of type of objects to export. Ex: "Ort,Alpiner gipfel,See"
+    -d : Optional. Display debug info about skipped features
+"""
+    try: 
+        opts, args = getopt.getopt(sys.argv[1:],"i:o:t:d") 
+    except getopt.GetoptError:
+        print help_msg
+        sys.exit()
+    for opt, arg in opts:
+        print opt
```
Comment (loicgasser) [inline scripts/kml_labels.py]: no need

File: scripts/kml_labels.py [hunk 2]
```diff
@@ -58,7 +59,55 @@ def escape_chars(txt):
...
+                sys.exit()
+        elif opt == "-d":
+            debug = True
+    
+
+    
+    if not file_path:
+        print('Input file %s does not exist' % file_path)
+        print help_msg
+        sys.exit()
```
Comment (loicgasser) [inline scripts/kml_labels.py]: why not do that earlier? same below

File: scripts/kml_labels.py [hunk 3]
```diff
@@ -109,8 +151,15 @@ def main():
                 layerid = float(feature['properties']['layerid'])
                 objektart = feature['properties']['objektart']
                 hoehe = float(feature['properties']['hoehe'])
-                if type(name) == unicode and type(uuid) == unicode and valid_coords(coordinates) and \
+                skip = True if not len(types) == 0 else False
+                for typee in types:
+                    if objektart == typee:
+                        skip = False
+                        break
```
Comment (loicgasser) [inline scripts/kml_labels.py]: replace all the above with
```python
skip = objektart in types
```

File: scripts/kml_labels.py [hunk 4]
```diff
@@ -58,7 +59,55 @@ def escape_chars(txt):
 
 
 def main():
-    save_folder = '/home/ltgal/data/swissnames/'
+    save_folder = '' 
+    file_path = ''
+    types = []
+    debug = False
+    help_msg =  """
```
Comment (loicgasser) [inline scripts/kml_labels.py]: you could also use dedent from textwrap.

File: scripts/kml_labels.py [hunk 5]
```diff
@@ -109,8 +151,15 @@ def main():
...
                 objektart = feature['properties']['objektart']
                 hoehe = float(feature['properties']['hoehe'])
-                if type(name) == unicode and type(uuid) == unicode and valid_coords(coordinates) and \
+                skip = True if not len(types) == 0 else False
+                for typee in types:
+                    if objektart == typee:
+                        skip = False
+                        break 
+                                         
+                if not skip and type(name) == unicode and type(uuid) == unicode and valid_coords(coordinates) and \
```
Comment (loicgasser) [inline scripts/kml_labels.py]: I know it's my code but let's use a more compact syntax.
```python
all([isinstance(name, unicode),  isinstance(objektart, unicode), isinstance(uuid, unicode), isinstance(layerid, float), isinstance(hoehe, float)])

**Comment by loicgasser** [comment]
LGTM some small comments

File: scripts/kml_labels.py (hunk 2 above)
Comment (oterral) [inline scripts/kml_labels.py]: because file_path is not set beofre.

File: scripts/kml_labels.py (hunk 3 above)
Comment (oterral) [inline scripts/kml_labels.py]: better indeed

**Comment by oterral** [comment]
@loicgasser fixed according to comment

**Pull request merged**